services/src/main/java/org/keycloak/protocol/oid4vc/model/ProofType.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.protocol.oid4vc.model;
+
+
+/**
+ * Enum to provide potential proof types for holder-binding
+ *
+ * @author <a href="https://github.com/wistefan">Stefan Wiedemann</a>
+ */
+public enum ProofType {
+
+    JWT("jwt"),
+    LD_PROOF("ld_proof");
```

Comment:
Is this `ld_proof` correct? I think it is `ldp_vp`

Reply:
fixed in [#efc83d5](https://github.com/keycloak/keycloak/pull/27931/commits/efc83d575b68430da840ad8cc53ce2eb0ffb887e)